warehouse/query-core/src/main/java/datawave/query/jexl/JexlASTHelper.java

```diff
@@ -107,20 +107,26 @@
     /**
      * Parse a query string using a JEXL parser and transform it into a parse tree of our RefactoredDatawaveTreeNodes. This also sets all convenience maps that
      * the analyzer provides.
-     * 
+     *
      * @param query
      *            The query string in JEXL syntax to parse
+     * @param preserveEscapes
+     *            True if escape characters should be left as-is, and not interpreted
      * @return Root node of the query parse tree.
      * @throws ParseException
      */
-    public static ASTJexlScript parseJexlQuery(String query) throws ParseException {
+    public static ASTJexlScript parseJexlQuery(String query, boolean preserveEscapes) throws ParseException {
         // Instantiate a parser and visitor
         Parser parser = new Parser(new StringReader(";"));
         
         String caseFixQuery = query.replaceAll("\\s+[Aa][Nn][Dd]\\s+", " and ");
         caseFixQuery = caseFixQuery.replaceAll("\\s+[Oo][Rr]\\s+", " or ");
         caseFixQuery = caseFixQuery.replaceAll("\\s+[Nn][Oo][Tt]\\s+", " not ");
         
+        if (preserveEscapes) {
+            caseFixQuery = caseFixQuery.replace("\\\\", "\\\\\\\\");
```

Comment:
replace every 2 with 4 instead?

Reply:
Check the latest commit.  I decided to move away from fixing the slashes here.